lib/authenticator.js

```diff
@@ -130,7 +130,7 @@ class Authenticator {
    *     app.use(passport.initialize({ userProperty: 'currentUser' }));
    *
    * @param {AuthenticatorInitializeOptions} options
-   * @returns {Function} middleware
+   * @returns {GenericCallback} middleware
    * @public
```

Comment:
Rather than suppressing the errors by using `GenericCallback`, as that is really only mean for callbacks which are truly arbitrary in form, I'd instead recommend temporarily disabling the `jsdoc/check-types` rule in `.eslintrc.*` so that these `Function` types can later be tied to specific `@callback` signatures for full documentation. I can do this in a subsequent commit if you prefer.

Reply:
lintfix changed those, let's leave them for now and add sort it out once we have jsdoc actually in use.